Makefile.am

```diff
@@ -132,6 +132,38 @@ CLEANFILES += $(eos_updater_avahi_in:%.in=%)
 
 dist_man8_MANS += eos-updater-avahi/docs/eos-updater-avahi.8
 
+# Flatpak install helper
+libexec_PROGRAMS += eos-updater-flatpak-installer/eos-updater-flatpak-installer
+
+eos_updater_flatpak_installer_eos_updater_flatpak_installer_CPPFLAGS = \
+	$(AM_CPPFLAGS) \
+	-DG_LOG_DOMAIN=\""eos-updater-flatpak-installer"\" \
+	$(NULL)
+eos_updater_flatpak_installer_eos_updater_flatpak_installer_CFLAGS = \
+	$(WARN_CFLAGS) \
+	-I$(top_srcdir)/src \
+	$(CODE_COVERAGE_CFLAGS) \
+	$(FLATPAK_CFLAGS) \
+	$(OSTREE_CFLAGS) \
+	$(GLIB_CFLAGS) \
+	$(GIO_CFLAGS) \
+	$(NULL)
+eos_updater_flatpak_installer_eos_updater_flatpak_installer_LDFLAGS = \
+	$(WARN_LDFLAGS) \
+	$(NULL)
+eos_updater_flatpak_installer_eos_updater_flatpak_installer_LDADD = \
+	$(CODE_COVERAGE_LIBS) \
+	$(top_builddir)/src/libeos-updater-dbus.la \
+	$(top_builddir)/libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la \
+	$(FLATPAK_LIBS) \
+	$(OSTREE_LIBS) \
+	$(GLIB_LIBS) \
+	$(GIO_LIBS) \
+	$(NULL)
+eos_updater_flatpak_installer_eos_updater_flatpak_installer_SOURCES = \
+	eos-updater-flatpak-installer/eos-updater-flatpak-installer.c \
```

Comment:
Please call it `eos-updater-flatpak-installer/main.c` (easier to type and tab-complete).